firebase.json

```diff
@@ -328,6 +328,7 @@
       { "source": "/go/nullable-cupertinothemedata-brightness", "destination": "https://docs.google.com/document/d/1qivq0CdkWGst5LU5iTLFUe_LTfLY84679-NxWiDgJXg/edit", "type": 301 },
       { "source": "/go/ondirtycallbacks", "destination": "https://docs.google.com/document/d/1Vk_QWC92fFGxx2oIrIjkCL0ZZFHxmrprlLedyQsnkus/edit?usp=sharing", "type": 301 },
       { "source": "/go/opengl-on-ios", "destination":"https://docs.google.com/document/d/1kvrb6HeTRN4noAKO82o6x-Ii61PW-BQ-8s9YNtLgosM", "type": 301 },
+      { "source": "/go/optimized-platform-view-layers", "destination": "https://docs.google.com/document/d/1YHwVz7-F03psEzHxByGka_lIFaDV5K90BJMbNxQeK4k/edit#", "type": 301 },
```

Comment:
Should this doc be public?

Reply:
My bad, the URL was incomplete. Can you retry?

Reply:
I tried opening the [url](https://docs.google.com/document/d/1YHwVz7-F03psEzHxByGka_lIFaDV5K90BJMbNxQeK4k/edit) in an incognito window and it doesn't work for me. It requires a login. I can't land until this opens in an incognito window.

Reply:
Did you try URL from [this commit](https://github.com/flutter/website/pull/7866/commits/10f9857cef7f7e51852ce08cba0b7bb3ba8b93f1)? It works for me in incognto mode. The one before (from previous commit, mentioned here), doesn't.